storage/url/url_test.go

```diff
@@ -70,7 +71,7 @@ func TestNew(t *testing.T) {
 				Prefix:    "key",
 				Delimiter: "/",
 			},
-			wantFilterRe: regexp.MustCompile(`^key.*$`).String(),
+			wantFilterRe: regexp.MustCompile(strutil.AddNewLineFlag(`^key.*$`)).String(),
```

Comment:
IMHO instead of using `strutil.AddNewLineFlag`, writing all regex will be much more clear (`(?s)^key.*$`). If I change the logic of strutil.AddNewLineFlag, tests will not fail. Not sure if this is something we want.